_rules/SC4-1-1-unique-id.md

```diff
@@ -11,20 +11,18 @@ test_aspects:
 
 authors:
 - Bryn Anderson
-- Jey Nandakumar
-- Kamyar Rasta
-- Wilco Fiers
+- Anne Thyme
 ---
 
 ## Test procedure
 
 ### Applicability
 
-Each test target has an `id` attribute that is unique within the [document context](https://github.com/auto-wcag/auto-wcag/pull/110/files#insert-link-here) of the element.
+Any element that has an `id` attribute within a document.
```

Comment:
Why "within a document". When is an element not "within a document"? What does that even mean?

Reply:
Probably refers to the "document context" in the Expectation. Since it's in the Expectation, however, there's no need to have anything like that here, so I'll remove it.